<?xml version="1.0"?>
<launch>
 
    <!-- Loading parameters-->
    <param name="obstacle_distance_threshold_" value="6.0"/>
<param name="field_of_view_angle_degrees_" value="180.0"/>
    
    <!--Generally start with a small bubble radius and increase further.-->
    <param name="safety_bubble_radius_" value="0.50"/>    <!--1.5 for levine hall, 3 for columbia-->
                                                          <!--0.5 for levine obstructed-->
    <param name="sliding_window_length_" value="5"/>
    <param name="safety_angle_" value="20.00"/> 
    <param name="buffer_angle_scaling_factor_" value="6.0"/>   <!--6.5 for columbia--> <!--3 worked for levine obstructed-->

    <!--Launching the node-->
    <node name="follow_the_gap_node" pkg="follow_the_gap" type="follow_the_gap_node" output="screen"/>

</launch>